otter/test/models/test_cass_models.py

```diff
@@ -411,10 +413,23 @@ def _responses(*args):
         self.policies = []
         self.mock_log = mock.MagicMock()
 
-        self.kz_client = mock.Mock()
-        self.lock = self.mock_lock()
-        self.kz_client.Lock.return_value = self.lock
-        self.kz_client.delete.return_value = defer.succeed('something else')
+        self.kz_client = ZKCrudModel()
+        self.kz_client.nodes = {"/locks/" + self.group_id: ("", 0)}
+
+        self.acquire_call = (True, 10, True)
+        self.release_call = None
+
+        def create_ZKLock(disp, path):
+            self.assertEqual(disp, "dispatcher")
+            self.lock = ZKLock("client", path)
+            self.lock.acquire_call = self.acquire_call
+            self.lock.release_call = self.release_call
+            return self.lock
+
+        from otter.models.cass import zk
+        self.patch(zk, "PollingLock", create_ZKLock)
```

Comment:
Speaking of checking correspondence between different things :).  The docstring for `ZKLock` hasn't even been updated to indicate that it's a mock of `PollingLock` yet, which makes me wonder if this `patch` is really correct.

Do you really need to mock out `PollingLock` like this?  Isn't all I/O already routed through this test's `dispatcher`, which won't do any Zookeeper I/O?